pkg/controller/volume/cache/actual_state_of_world.go

```diff
@@ -132,7 +134,7 @@ type actualStateOfWorld struct {
 // believes to be succesfully attached to a node it is managing.
 type attachedVolume struct {
```

Comment:
It is a little unexpected to see two different types with basically the same name in a package, one exported, one not.  Suggest that we either rename this type somehow or disambiguate with a comment, but would strongly prefer rename.